src/headers/tomcrypt_hash.h

```diff
@@ -113,6 +113,30 @@ struct chc_state {
 };
 #endif
 
+#ifdef LTC_BLAKE2S
+struct blake2s_state {
+    ulong32 h[8];
+    ulong32 t[2];
+    ulong32 f[2];
+    unsigned char buf[2 * 64];
+    ulong32 curlen;
```

Comment:
also these types (`curlen, outlen, last_node`) could be the same type as in blake2b or not? (or in blake2b the same as here, whichever makes most sense, at least it should be consistent)